release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Remove deprecated field for submissions
```

Comment:
Can we change this from Remove -> Removed

Reply:
Done